solutions/07b/single-chat.js

```diff
@@ -11,7 +11,7 @@ feed.append({
   nickname: 'cat-lover',
   text: 'hello world',
   timestamp: '2018-11-05T14:26:000Z' // new Date().toISOString()
-}, function (err, seq) {
+}, function(err, seq) {
```

Comment:
I noticed that you changed the formatting throughout of
```js
function (args)
```
to
```js
function(args)
```
Can you please maintain this convention?

Reply:
If you're wondering where it comes from, it is standard style -- the `standard` cli tool has a feature that automatically format it for you in this style

Reply:
Reformatted d021a4f4b8aa7e6c61c06206af92b30556d519bc